argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -98,12 +98,13 @@ spec:
     targetRevision: main
 ---
 apiVersion: argoproj.io/v1alpha1
-kind: Application
+kind: ApplicationSet
 metadata:
   name: enterprise-contract
-  finalizers:
-    - resources-finalizer.argocd.argoproj.io
+  namespace: openshift-gitops
```

Comment:
Why is this needed? (It is not needed for Applications)

Reply:
Removed